ingestion_tools/scripts/tests/s3_import/test_gains.py

```diff
@@ -8,7 +8,7 @@
 from importers.run import RunImporter
 from mypy_boto3_s3 import S3Client
 from standardize_dirs import IMPORTERS
-from tests.s3_import.test_annotations_import import list_dir
+from tests.s3_import.util import list_dir
```

Comment:
nit: looks like this is imported and not used?

Reply:
It is used in the file, example [line](https://github.com/chanzuckerberg/cryoet-data-portal-backend/pull/118/files#diff-9dcb05a762b5ed051838d1a9cb70bc1b1acb4ada435d3c2d0c334647cc03d901R45).